r2/r2/controllers/api.py

```diff
@@ -885,6 +908,10 @@ def POST_unfriend(self, nuser, iuser, container, type):
         if new and type in self._sr_friend_types:
             ModAction.create(container, c.user, action, target=victim)
 
+        if was_mod:
+            send_mod_permission_message(container, c.user, victim,
+                                        was_mod_perms, demod=True)
```

Comment:
I do have concerns about abuse.

Possibly with two processes spanning two accounts on a mod list.

https://www.reddit.com/r/ideasfortheadmins/comments/42h0l3/get_a_notification_when_you_are_demodded_from_a/czaege0

Unless I'm not seeing something crucial here like the API limits being a limiting factor...